internal/command/testdata/test/with_verify_module/main.tftest.hcl

```diff
@@ -0,0 +1,23 @@
+variables {
+  id = "resource"
+  value = "Hello, world!"
+}
+
+run "test" {
+}
+
+run "verify" {
+  module {
+    source = "./verify"
+  }
+
+ assert {
```

Comment:
The formatting seems to be slightly off here?

Reply:
You are sharp 👁️ 
Fixed https://github.com/opentofu/opentofu/pull/1043/commits/d3d39f2b3b743ffadfca90c8172d2a536f8363dc